File: tests/Generator/resource/csharp/default/Location.cs
using System.Text.Json.Serialization;

/// <summary>
/// Location of the person
/// </summary>
public class Location
{
    [JsonPropertyName("lat")]
    public double? Lat { get; set; }

    [JsonPropertyName("long")]
    public double? Long { get; set; }

}

File: tests/Generator/resource/java/default/Location.java

import com.fasterxml.jackson.annotation.JsonGetter;
import com.fasterxml.jackson.annotation.JsonSetter;

/**
 * Location of the person
 */
public class Location {
    private Double lat;
    private Double _long;

    @JsonSetter("lat")
    public void setLat(Double lat) {
        this.lat = lat;
    }

    @JsonGetter("lat")
    public Double getLat() {
        return this.lat;
    }

    @JsonSetter("long")
    public void setLong(Double _long) {
        this._long = _long;
    }

    @JsonGetter("long")
    public Double getLong() {
        return this._long;
    }
}

